pycbc/workflow/datafind.py

```diff
@@ -419,14 +414,27 @@ def setup_datafind_runtime_cache_multi_calls_perifo(cp, scienceSegs,
 
             # Sometimes the connection can drop, so try a backup here
             try:
```

Comment:
Is this try/except still needed, as the connection is no longer kept open?

Reply:
I don't know. The old code never *reset* the connection, but simply made the call again. This may not be needed, but I think my preferred solution is just to remove this function (see issue added recently), so don't really want to worry about this too much.